masonry/src/widget/widget_ref.rs

```diff
@@ -169,7 +168,7 @@ impl<'w> WidgetRef<'w, dyn Widget> {
     pub fn find_widget_at_pos(&self, pos: Point) -> Option<WidgetRef<'_, dyn Widget>> {
         let mut innermost_widget = *self;
 
-        if !self.state().window_layout_rect().contains(pos) {
+        if !self.ctx.widget_state.window_layout_rect().contains(pos) {
```

Comment:
Is there a reason you're calling `window_layout_rect` on the state instead of the context? It would seem better abstracted to call it on the context.

Reply:
Good catch.